go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/opencontainers/runc
 
-go 1.22
+go 1.22.4
```

Comment:
Could you add a comment line to explain why 1.22.0-1.22.3 are purposely omitted?

Reply:
Done, thanks!